app/workers/bgs/submit_form674_job.rb

```diff
@@ -11,6 +11,7 @@ class Invalid674Claim < StandardError; end
 
     sidekiq_options retry: false
 
+    # rubocop:disable Metrics/MethodLength
```

Comment:
can you chuck a comment here explaining this is only due to flipper extending the method length?